Profiling Docs overhaul update (#7803)
DataDog/documentation: content/en/tracing/profiling/troubleshooting.md

---
title: Troubleshooting
kind: Documentation
further_reading:
    - link: 'tracing/profiling/getting_started'
      tag: 'Documentation'
      text: 'Enable continuous profiling for your application.'
    - link: 'tracing/profiling/trace_profile'
      tag: 'Documentation'
      text: 'Connect traces and profiles.'
    - link: 'tracing/profiling/code_level_metrics'
      tag: 'Documentation'
      text: 'View code level metrics.'  
    - link: 'tracing/profiling/search_profiles'
      tag: 'Documentation'
      text: 'Learn more about available profile types.'
    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
      tags: 'Blog'
      text: 'Introducing always-on production profiling in Datadog.'
---

> In case you have done all the necessary configuration steps and do not see profiles in the [profile search page](#search-profiles), turn on [debug mode][1] and [open a support ticket][2] with debug files and the following information:

Reviewer: ```suggestion
If you've configured the profiler and don't see profiles in the [profile search page](#search-profiles), please turn on [debug mode][1] and [open a support ticket][2] with debug files and the following information:
```
